modules/distribution/src/sso/views/register_new_user.jag

```diff
@@ -0,0 +1,200 @@
+<%
+/*
+ *  Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+var configs = require('/config/sso.js').config();
+var e = request.getParameter('authFailure');
+
+var log = new Log();
+var relyingParty = request.getParameter('relyingParty');
+
+if(!request.isSecure()){
+    response.content = { 'authFailure' : e , 'relyingParty' : relyingParty };
+    response.sendRedirect(configs.server.https + request.getRequestURI());
+}
+
+//TODO: this should be fixed properly
+if (relyingParty instanceof Array) {
+    relyingParty = relyingParty.pop();
+}
+var signInUrl;
+
+if (relyingParty == 'publisher') {
```

Comment:
Single quote is the standard.
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings